src/styles/tailwind.css

```diff
@@ -2,6 +2,27 @@
 @import "tailwindcss/components";
 @import "tailwindcss/utilities";
 
+/* this layer for overwrite Obsidian default css style(file: app.css) and theme css style */
+@layer components {
+  .modal-container input.input-reset,
+  input[type="text"].input-reset,
+  input[type="password"].input-reset,
+  input[type="number"].input-reset {
+    @apply h-9 !min-w-[50px] border border-solid rounded-md bg-transparent px-3 py-1 md:text-base text-sm shadow-sm transition-colors;
+    @apply focus-visible:shadow-sm focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring;
+  }
+
+  .modal-container button.select-trigger-reset {
+    @apply h-9 border border-solid rounded-md bg-dropdown px-3 py-1 text-base md:text-sm shadow-sm transition-colors;
+    @apply focus-visible:shadow-sm focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring;
```

Comment:
is there any reasons why we don't add these classes directly to the component? (e.g. make them part of SelectPrimitive.Trigger). Same for the others